apps/django_app.py

```diff
@@ -18,28 +18,22 @@
     compat_url(r"^$", lambda r: redirect("/vulnpy"))
 ] + vulnerable_urlpatterns
 
-filename = os.path.basename(os.path.splitext(__file__)[0])
+if not settings.configured:
+    settings.configure(
+        **{
+            "ROOT_URLCONF": "django_app"
+            if __name__ == "__main__"
+            else "apps.django_app",
+            "ALLOWED_HOSTS": ["localhost", "127.0.0.1", "[::1]"],
```

Comment:
had no idea what ` "[::1]"]` was for but it was already there

Reply:
it's the IPv6 version of 127.0.0.1!